server.js

```diff
@@ -9,19 +9,15 @@ var https = require('https');
 var request = require('request');
 var tmp = require('tmp');
 
-var app = express();
 
 // Different headers can be pushed depending on data format
 // to allow for changes with backwards compatibility
-var header = {
+var UP1_HEADERS = {
     v1: new Buffer("UP1\0", 'binary')
 }
 
-var config = JSON.parse(fs.readFileSync('./server.conf'));
-
-app.use('', express.static('public'));
-
-app.post('/up', function(req, res) {
+function handleupload(req, res) {
```

Comment:
Change to handle_upload and use snake case for other functions and variables (I might be using camel case in places, which is inconsistent)